locale/ru-ru.js

```diff
@@ -26,7 +26,7 @@ export default {
     labelMaxTotalFileSizeExceeded: 'Превышен максимальный размер',
     labelMaxTotalFileSize: 'Максимальный размер файла: {filesize}',
     labelFileTypeNotAllowed: 'Файл неверного типа',
```

Comment:
`fileValidateTypeLabelExpectedTypes ` was not translated to russian at all. It was portuguese.

I've changed it to be: `Expecting {allButLastType} or {lastType}`